content/docs/learning-guide/tutorials/entities-and-prefabs/entity-and-prefab-basics.md

```diff
@@ -0,0 +1,216 @@
+
+The sections in this topic demonstrate the basics of working with entities, prefabs, and prefab instances.
+
+## Create an entity
+
+You can create an entity with either of the following methods:
+
+* In Entity Outliner or the viewport, **right-click** and choose **Create entity** from the context menu.
+* With Entity Outliner focused, press **CTRL+ALT+N**.
+
+![Creating a new entity in Entity Outliner.](/images/learning-guide/tutorials/entities-and-prefabs/create-entity.png)
+
+To create an entity with an asset as its basis, **drag** an asset such as a `.azmodel` or a `.actor` from **Asset Browser** into Entity Outliner or the viewport. In the following video, a `.azmodel` is dragged into the viewport from Asset Browser, and a new entity with a **Mesh** component is created in the level.
+
+{{< video src="/images/learning-guide/tutorials/entities-and-prefabs/create-entity.mp4" info="Drag and drop to create an entity." autoplay="true" loop="true" width="900" >}}
+
+## Create a prefab
+
+Prefabs are saved to disk and allow you to easily reuse objects by instantiating them in a level or spawning them at runtime. Prefabs are created from entities with the following steps:
+
+1. In Entity Outliner, **right-click** on an entity and choose **Create Prefab...** from the context menu.
+1. In the **Save As…** window, navigate to your project’s `Prefabs` directory, supply a name for the prefab, and choose **Save**.
+
+![Creating a prefab from an entity in Entity Outliner.](/images/learning-guide/tutorials/entities-and-prefabs/create-prefab.png)
+
+The prefab file is saved to disk and an instance of the prefab replaces the entity in **O3DE Editor**. The prefab instance displays the name of the instance on the left next to the {{< icon "prefab.svg" >}} prefab icon. The name of the prefab file is displayed on the right in parenthesis. In the following image, notice that the level itself is a type of prefab.
+
+![A new prefab instance in Entity Outliner.](/images/learning-guide/tutorials/entities-and-prefabs/prefab-instanced.png)
+
+### Create a prefab from a collection of entities and prefab instances
+
+Complex prefabs might require multiple entities and prefab instances. You can create a prefab from a combination of entities and prefab instances with the following steps:
+
+1. Select the entities and prefab instances you want included in the new prefab. You can select multiple entities and prefab instances by holding **SHIFT** and clicking on each entity and prefab instance. Alternatively, you can **drag** a selection box around a group of entities and prefab instances in Entity Outliner or in the viewport.
+1. In Entity Outliner, **right-click** one of the selected entities or prefab instances and choose **Create Prefab...** from the context menu.
+1. In the **Save As…** window, navigate to your project’s `Prefabs` directory, supply a name for the prefab, and choose **Save**.
```

Comment:
same as above. This can be any directory, not just 'Prefabs'

Reply:
done